bin/commands/.help

```diff
@@ -1,3 +1,5 @@
-A command line utility helps you managing Zowe instance.
+Description
```

Comment:
After removing `Synopsis` from `.help`, i guess we don't need this `Description` line. What do you think?

Reply:
Right, Fixed it.